CHANGELOG.md

```diff
@@ -9,6 +9,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- The metric registration is simplified. The user can now directly write a score function within
+  its script, and directly register it by specifying the right dependencies and permissions.
```

Comment:
Nitpick:
```suggestion
  their script, and directly register it by specifying the right dependencies and permissions.
```

I'm not sure, but I think "its script" refers to the user, and referring to humans as "it" is impolite. But maybe it's the function's script?

Reply:
I think it is the metric registration's script, but in case it's the user's it would be "his" not "their" as it's singular :)

Reply:
https://strategiesforparents.com/its-or-their-when-to-use-each-possessive-pronoun/ their refer to one or several, and indeed, it is better if we refer to people ! Thanks for this little english gramar course 😄